railties/lib/rails/mailers_controller.rb

```diff
@@ -12,6 +12,7 @@ class Rails::MailersController < Rails::ApplicationController # :nodoc:
   helper_method :part_query, :locale_query
 
   content_security_policy(false)
+  content_security_policy_report_only(false)
```

Comment:
This suggests that upgrading apps will need to add this method to disable CSP-Report-Only?

Reply:
Good call. This isn't necessary. There is no default report only policy.

Reply:
Removed.